src/Okta.Sdk.IntegrationTests/InlineHooksClientShould.cs

```diff
@@ -0,0 +1,340 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    public class InlineHooksClientShould
+    {
+        private const string SdkPrefix = "dotnet_sdk";
+        private const string InlineHookVersion = "1.0.0";
+        private const string InlineHookType = "com.okta.oauth2.tokens.transform";
+
+        private static readonly InlineHookChannel TestInlineHookChannel = new InlineHookChannel
+        {
+            Type = "HTTP",
+            Version = "1.0.0",
+            Config = new InlineHookChannelConfig
+            {
+                Uri = "https://www.example.com/inlineHook",
+                Headers = new List<IInlineHookChannelConfigHeaders>
+                {
+                    new InlineHookChannelConfigHeaders
+                    {
+                        Key = "X-Test-Header",
+                        Value = "Test header value",
+                    }
+                },
+                AuthScheme = new InlineHookChannelConfigAuthScheme
+                {
+                    Type = "HEADER",
+                    Key = "Authorization",
+                    Value = "Test-Api-Key",
+                }
+            }
+        };
+
+        private static readonly InlineHookChannel UpdatedTestInlineHookChannel = new InlineHookChannel
+        {
+            Type = "HTTP",
+            Version = "1.0.0",
+            Config = new InlineHookChannelConfig
+            {
+                Uri = "https://www.example.com/inlineHookUpdated",
+                Headers = new List<IInlineHookChannelConfigHeaders>
+                {
+                    new InlineHookChannelConfigHeaders
+                    {
+                        Key = "X-Test-Header",
+                        Value = "Test header value updated",
+                    }
+                },
+                AuthScheme = new InlineHookChannelConfigAuthScheme
+                {
+                    Type = "HEADER",
+                    Key = "Authorization",
+                    Value = "Test-Api-Key-Updated",
+                }
+            }
+        };
+        
+        public InlineHooksClientShould()
+        {
+            DeleteAllInlineHooks().Wait();
+        }
+        
+        [Fact]
+        public async Task CreateInlineHook()
+        {
+            var testClient = TestClient.Create();
+            var testInlineHookName = $"{SdkPrefix}:{nameof(CreateInlineHook)} Test Inline Hook Name";
+
+            var createdInlineHook = await testClient.InlineHooks.CreateInlineHookAsync(new InlineHook
```

Comment:
Nit: Let's check we are following StyleCop recommended format here.

Reply:
Updated.